src/Utility.ts

```diff
@@ -214,4 +214,26 @@ export namespace Utility {
         }
         return javaPath ? javaPath + '/bin/java' : 'java';
     }
+
+    export function getJavaEnvironment(options?: child_process.SpawnOptions): child_process.SpawnOptions {
```

Comment:
No need to bring `options` in. 

`export function getCustomEnv(): { [key: string]: string }` would be simple and clear.